src/lib/github/getFilesOnInstall.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Gets files that have changed in the last  commits
+ */
+import { Context } from "./types";
+import { getBasicRepoProps } from "./utils";
+
+export interface ModifiedFile {
+  name: string;
+  downloadUrl: string;
+  htmlUrl: string;
+}
+
+export default async function getFiles(context: Context): Promise<ModifiedFile[]> {
+  const { owner, repo } = getBasicRepoProps (context);
+  const octokit = context.github;
+  const committedArray = await octokit.repos.getCommit({owner, repo});
+  const changedFilesArray = committedArray.map( async (obj) => {
+    const sha = obj.sha;
+    return await octokit.repos.getCommit({owner, repo, sha});
+  });
+  // const arrayUniq = R.uniq(obj => obj.files[0].filename, changedFilesArray);
+  // wanted to remove duplicate files but realized files may have different todos
+  const modifiedFiles: Array<Promise<ModifiedFile>> =
```

Comment:
I think we already have a similar function, would have been better to turn this into a util function and use it both modules.

Reply:
want to use the some approach so that wen we get the last commits we follow the same approach and just remove duplicates